tests/PHPStan/Rules/PhpDoc/data/value-of-enum.php

```diff
@@ -0,0 +1,26 @@
+<?php // lint >= 8.1
+
+declare(strict_types=1);
+
+namespace ValueOfEnum;
+
+enum Country: string
+{
+	case NL = 'The Netherlands';
+	case US = 'United States';
+}
+
+class Foo {
+	/**
+	 * @param value-of<Country> $countryName
+	 */
+	function hello(string $countryName): void
```

Comment:
Please also add a test that fails, e.g. string typehint + int-backed enum or similar.